main.rb

```diff
@@ -1,13 +1,180 @@
 require_relative 'person'
 require_relative 'student'
 require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+require 'colorize'
```

Comment:
### Optional
- [ ] It would be better to install any gem we plan to use in order to avoid errors. Nice job with the use of the colorize gem

Reply:
I wanted to, But didn't quit know how I already used `gem install colorize` I thought it would be just like Node package manager will be added automatically, Could you kindly help me with that if there is a way to add it automatically

Reply:
Yeah sure, kindly add `gem colorize` in the Gemfile and run `bundle install`. Then you can then require it in your application. Hope that helps and not late :smile: